getting_started/installation.adoc

```diff
@@ -4,9 +4,9 @@
 .Disclaimer
 [WARNING]
 ====
-Please note that LibrePCB is a very young software and thus (probably)
-isn't as mature as other EDA tools are. Use LibrePCB at your own risk,
-accepting that we don't take responsibility for any implications caused
+Please note that LibrePCB is a very early software and thus
```

Comment:
"a very early software" still sounds a bit strange to me. Maybe someone has an idea for better wording?

Reply:
Yes, both are unusual. I would suggest the phrase 'librePCB is still in early development'